new-docs/source/tutorial/1-getting-started.ipynb

```diff
@@ -0,0 +1,340 @@
+    "\n",
+    "* import the class from the `pydra.tasks.*` package it is in\n",
+    "* instantiate it with appropriate parameters\n",
+    "* \"call\" resulting object (i.e. `my_task(...)`) to execute it as you would a function \n",
+    "\n",
+    "To demonstrate with an example of loading a JSON file with the\n",
+    "`pydra.tasks.common.LoadJson` task, we first create an example JSON file to test with"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can load the JSON contents back from the file using the `LoadJson` task definition\n",
+    "class"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import the task definition\n",
+    "from pydra.tasks.common import LoadJson\n",
+    "\n",
+    "# Instantiate the task definition, providing the JSON file we want to load\n",
+    "load_json = LoadJson(file=json_file)\n",
+    "\n",
+    "# Run the task to load the JSON file\n",
+    "outputs = load_json()\n",
+    "\n",
+    "# Access the loaded JSON output contents and check they match original\n",
+    "assert outputs.out == JSON_CONTENTS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Iterating over inputs\n",
+    "\n",
+    "It is straightforward to apply the same operation over a set of inputs using the `split()`\n",
+    "method. For example, if we wanted to re-grid all the NIfTI images stored in a directory,\n",
+    "such as the sample ones generated by the code below"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then we can by importing the `MrGrid` shell-command task from the `pydra-mrtrix3` package\n",
+    "and run it over every NIfTI file in the directory using the `TaskDef.split()` method"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pydra.tasks.mrtrix3.v3_0 import MrGrid\n",
+    "\n",
+    "# Instantiate the task definition, \"splitting\" over all NIfTI files in the test directory\n",
+    "# by splitting the \"input\" input field over all files in the directory\n",
+    "mrgrid = MrGrid(operation=\"regrid\", voxel=(0.5,0.5,0.5)).split(in_file=nifti_dir.iterdir())\n",
+    "\n",
+    "# Run the task to resample all NIfTI files\n",
+    "outputs = mrgrid()\n",
+    "\n",
+    "# Print the locations of the output files\n",
+    "pprint(outputs.out_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is also possible to iterate over inputs in pairs/n-tuples. For example, if you wanted to use\n",
+    "different voxel sizes for different images, both the list of images and the voxel sizes\n",
+    "are passed to the `split()` method and their combination is specified by a tuple \"splitter\"\n",
```

Comment:
probably it should be unified how the splitters are called, since on the main page for splitter, it is called scalar

Reply:
I think I would vote for "pairwise" or something like that (although I suppose you could have N>2 "pairs")

Reply:
maybe "zipped" to match the Python function?